frontend/test/stores/appointment-store.test.js

```diff
@@ -86,6 +86,12 @@ describe('Appointment Store', () => {
     expect(apmt.pendingAppointments.length).toBe(1);
   });
 
+  test('timezone', async () => {
+    const apmt = useAppointmentStore();
+    await apmt.fetch(createFetch({ baseUrl: API_URL }));
+    expect(apmt.appointments[0].slots[0].start.toISOString()).toBe('3000-01-01T09:00:00.000Z');
```

Comment:
Is this sufficient? I think this only tests the fallback to tz.guess which is UTC in the testing environment. Any idea on how to simulate the users timezone here?

Reply:
https://day.js.org/docs/en/timezone/set-default-timezone this might work? I haven't looked at the code. If not then maybe we should start looking at other date libraries 🙀

Reply:
We could also adjust the github workflow runner to set a timezone. That might be a good fallback solution hmmm.

Reply:
Setting a default timezone after the dj objects are initialize won't have any effect, so I think your idea with the gh workflow runner timezone is the best/least effort solution. I added a hint in the code. Thank you for your good thoughts on this!

Reply:
Sounds good, we can probably fix this later!